### DISCUSSION TITLE
I don't know how to setup ESLint to suit my issue.

### DISCUSSION THREAD

**Discussion by AliaNeedsHelp** [codesandbox/codesandbox-client#5298] [General]
I'm making a simulation game using Matter.js and P5.js, but I don't know what to put in which files. Could anyone help me?

**Comment by SaraVieira** [comment]
Hey!

We don't really support changing eslint config :/

You can remove some errors by adding lines like

```
// eslint-disable-next-line
```

**Comment by AliaNeedsHelp** [comment]
Basically, even with the p5.js template, It's saying that p5.js-native
variables/functions are not defined. Also, I am completely new to ESLint
and npm.

**Reply by smhutch** [reply]
@AliaNeedsHelp could you share a link to one of your sandboxes please? I'll try to share some suggestions to reduce these errors. Thanks!
